doc/imported_samples/marking_criteria.yml

```diff
@@ -0,0 +1,36 @@
+Quality of Writing:
```

Comment:
Specify a `type:` here. You can also leave a comment in the YML file saying that rubric is the default.